AFQ/tasks/data.py

```diff
@@ -23,12 +23,28 @@
 DIPY_GH = "https://github.com/dipy/dipy/blob/master/dipy/"
 
 
-outputs = [
-    "data", "gtab", "img", "b0_file", "masked_b0_file", "dti_tf",
-    "dti_params_file", "dki_tf", "dki_params_file", "csd_params_file",
-    "pmap_file", "dti_fa_file", "dti_cfa_file", "dti_pdd_file",
-    "dti_md_file", "dki_fa_file", "dki_md_file", "dki_awf_file",
-    "dki_mk_file", "brain_mask_file"]
+outputs = {
+    "data": """DWI data as an ndarray for selected b values.
+    If patch2self is used, it is already applied to this data""",
+    "gtab": """a DIPY GradientTable with all the gradient information""",
+    "img": """unaltered DWI data in a Nifti1Image""",
+    "b0_file": """mean b0 file""",
+    "masked_b0_file": """mean b0 after applying the brain mask file""",
+    "dti_tf": """DTI TensorFit""",
+    "dti_params_file": """parameters for the DTI fit file""",
+    "dki_tf": """DKI DiffusionKurtosisFit""",
+    "dki_params_file": """parameters for the DKI fit file""",
+    "csd_params_file": """parameters for the CSD fit file""",
+    "pmap_file": """anisotropic power map file""",
+    "dti_fa_file": """DTI fractional anisotropy file""",
+    "dti_cfa_file": """DTI color fractional anisotropy file""",
+    "dti_pdd_file": """DTI PDD file""",
+    "dti_md_file": """DTI mean diffusivity file""",
+    "dki_fa_file": """DKI fractional anisotropy file""",
+    "dki_md_file": """DKI mean diffusivity file""",
+    "dki_awf_file": """DKI axonal water fraction file""",
+    "dki_mk_file": """DKI mean kurtosis file""",
+    "brain_mask_file": """brain mask file"""}
```

Comment:
I think we could put a little bit more information here. 

For example: `"""full path to a nifti file containing a brain mask"""`

Or: `"""full path to a nifti file containing DKI mean diffusivity"""`